### PR TITLE
gRPC tutorial: Clarify random assigned server port

### LINKED ISSUE #25253: Removed hard coded port 7042
Navigates to http://localhost:port, such as http://localhost:7042 is OK, but take out the rest of port 7042. See https://github.com/dotnet/AspNetCore.Docs/discussions/24514

Tell folks to record the port. Later on tell them where to get the port.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 1d298a6c-8ddc-dbc8-d219-0fa7138442c2
* Version Independent ID: b96be346-958a-cfcd-d80a-c352d64223e8
* Content: [Create a .NET Core gRPC client and server in ASP.NET Core](https://docs.microsoft.com/en-us/aspnet/core/tutorials/grpc/grpc-start?view=aspnetcore-6.0&tabs=visual-studio)
* Content Source: [aspnetcore/tutorials/grpc/grpc-start.md](https://github.com/dotnet/AspNetCore.Docs/blob/main/aspnetcore/tutorials/grpc/grpc-start.md)
* Product: **aspnet-core**
* Technology: **aspnetcore-tutorials**
* GitHub Login: @JamesNK
* Microsoft Alias: **jamesnk**

### PR DESCRIPTION
**Pull request by wadepickett**
[Internal Review](https://review.docs.microsoft.com/en-us/aspnet/core/tutorials/grpc/grpc-start?view=aspnetcore-6.0&branch=pr-en-us-25397&tabs=visual-studio)

Fixes #25253

Clarified where to find the server localhost port and what line of code to update. 
Removed references of the example port number in the text.

I did leave the port number in the client code example, which needs to be able to compile, and needs to work with the port setting in the example server app.

### CODE REVIEW

File: aspnetcore/tutorials/grpc/grpc-start.md [hunk 1]
```diff
@@ -75,11 +75,11 @@ The tutorial assumes familiarity with VS Code. For more information, see [Gettin
  
 [!INCLUDE[](~/includes/run-the-app6.0.md)]
 
-The logs show the service listening on `https://localhost:7042`.
+The logs show the service listening on `https://localhost:<port>`, where `<port>` is the localhost port number randomly assigned when the project is created and set in the *Properties\launchSettings.json.
```
Comment (serpent5) [inline aspnetcore/tutorials/grpc/grpc-start.md:78]:
```suggestion
The logs show the service listening on `https://localhost:<port>`, where `<port>` is the localhost port number randomly assigned when the project is created and set in `Properties\launchSettings.json`.
```

File: aspnetcore/tutorials/grpc/grpc-start.md [hunk 2]
```diff
@@ -219,9 +219,11 @@ dotnet add GrpcGreeterClient.csproj package Grpc.Tools
 
 * Build the client project to create the types in the `GrpcGreeterClient` namespace. The `GrpcGreeterClient` types are generated automatically by the build process.
 
-* Update the gRPC client `Program.cs` file with the following code:
+* Update the gRPC client `Program.cs` file with the following code.
 
-  [!code-csharp[](~/tutorials/grpc/grpc-start/sample6/GrpcGreeterClient/Program.cs?name=snippet2)]
+  [!code-csharp[](~/tutorials/grpc/grpc-start/sample6/GrpcGreeterClient/Program.cs?name=snippet2&highlight=6)]
+
+* In the preceding highlighted code, replace the localhost port number `7042` with the `HTTPS` port number specified in *Properties\launchSettings.json of the `GrpcGreeter` service project.
```
Comment (serpent5) [inline aspnetcore/tutorials/grpc/grpc-start.md:226]:
```suggestion
* In the preceding highlighted code, replace the localhost port number `7042` with the `HTTPS` port number specified in `Properties\launchSettings.json` within the `GrpcGreeter` service project.
```

**Review by serpent5 — COMMENTED** [review]
A couple of suggestions to stay consistent with the recent switch from `*...*` to `` `...` `` for filenames.

File: aspnetcore/tutorials/grpc/grpc-start.md (hunk 1 above)
Comment (wadepickett) [reply aspnetcore/tutorials/grpc/grpc-start.md:78]: Thanks Kirk!  I'm a little embarrassed I made that mistake.  Thanks for catching it!

**Review by serpent5 — APPROVED** [review]

File: aspnetcore/tutorials/grpc/grpc-start.md (hunk 1 above)
Comment (serpent5) [reply aspnetcore/tutorials/grpc/grpc-start.md]: Ha, no worries. I'm primed to spot these after the monster update I did in this space recently.

**Pull request merged**